packages/react/src/components/Storage/FileUploader/UploadTracker/UploadTracker.tsx

```diff
@@ -56,7 +57,7 @@ export function UploadTracker({
   const showEditButton =
     fileState === FileState.INIT ||
     (fileState === FileState.ERROR &&
-      errorMessage === translate('Extension not allowed'));
+      errorMessage === displayText.extensionNotAllowed);
```

Comment:
This has nothing to do with your PR but it's kind of weird to me that we would check against the `errorMessage` for a specific value to determine if we should render the edit button. Maybe we should have more diverse error states on `FileState`, cc @reesscot

Reply:
Yea I had a note of this too.